### ISSUE TITLE
Node stuck on Gathering information...

### ISSUE THREAD

**Issue by jeremyawhite** [haydendonald/blackmagic-atem-nodered#3]
First, let me add my thanks for this node. I had a flow mostly working, but I was getting multiple messages with each change of Program/Preview inputs. Since I was using an older version, I decided to upgrade to 2.0. Since then, it's just gathering information. It won't send out messages for any events.

**Comment by haydendonald** [comment]
Hi, thank you for your question. 
What version did you happen to come from if you know? 
Another issue could be from the ATEM it's self so what version is your ATEM running, i've tested with the version 8.0 but it should work with versions down to 7.0.

Finally with the multiple messages, was this happening every time or randomly on deploy? I know there is an issue where the ATEM sends it's initial information differently sometimes on reconnect. Which i haven't been able to figure out yet so sometimes it thinks that the initial data is changes.

If possible please try pasting the code below into your flow and let me know how it goes!
```
[{"id":"253b5cda.9c0af4","type":"atem-atem","z":"4bccca49.11fcc4","name":"Test (Update my ip please)","network":"f4bfa39f.840a5","outputMode":"all","sendInitialData":"yes","sendStatusUpdates":"yes","x":540,"y":340,"wires":[["7d0b64ec.f28bfc"]]},{"id":"7d0b64ec.f28bfc","type":"debug","z":"4bccca49.11fcc4","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":770,"y":340,"wires":[]},{"id":"f4bfa39f.840a5","type":"atem-network","z":"","name":"Test","ipAddress":"10.0.0.1"}]
```

Thankyou!

**Comment by haydendonald** [comment]
Hi again, i have pushed an update (version 2.1.0) which may fix your issue!

**Comment by jeremyawhite** [comment]
I think I remember an 8 in the version, and it was probably pretty old. In the programInput command payload there was a data[0].inputNumber. I've tried up through 1.8.1 and haven't see that again. Perhaps my installation needs a check since the palette reports 1.2.8 (the oldest version I started on my quest to find my previous version).  My ATEM Software Control is version 7.2, and I don't think I've updated the hardware.

I updated the IP and pasted your nodes in after bodging your 2.1.0 changes in by hand. I restarted node-red, cleared my debug window and redeployed. In the first second, it reports connecting, disconnected ("Could not connect: The ATEM reported that it's full"), retries..connecting, disconnected. The next second it's connecting and initializing. That's where it stays. I deleted my original ATEM node and closed the software control, so there might be 1 connection in addition to the new node.

After installing 1.8.1 (I think) it seems to be working again. I'll try a clean install to verify.
